app/src/main/java/org/fossasia/openevent/general/order/OrdersUnderUserFragment.kt

```diff
@@ -29,6 +29,7 @@ import timber.log.Timber
 
 const val ORDERS: String = "orders"
 const val LAUNCH_TICKETS: String = "LAUNCH_TICKETS"
+const val SNACKBAR_MESSAGE: String = "SNACKBAR_MESSAGE"
```

Comment:
We need to remove duplication. What's the use of refactoring to a constant if you are going to write it in every file it is used